packages/react/src/components/link/Link.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { axe } from 'jest-axe';
+
+import { Link } from './Link';
+
+describe('<Link /> spec', () => {
+  it('renders the component', () => {
+    const { asFragment } = render(<Link href="https://hds.hel.fi">Test link</Link>);
+    expect(asFragment()).toMatchSnapshot();
+  });
+  it('should not have basic accessibility issues', async () => {
+    const { container } = render(<Link href="https://hds.hel.fi">Test link</Link>);
+    const results = await axe(container);
+    expect(results).toHaveNoViolations();
+  });
+});
```

Comment:
It would be nice to have a test for composing external links since there seem to be lots of logic with adding correct attributes and aria-labels

Reply:
A snapshot test for external link would be enough, if it is easier to implement

Reply:
- [ ] Add a test for composing external links